src/js/react/apps/linkedevents/components/TopicsFilter.tsx

```diff
@@ -0,0 +1,46 @@
+import { Select } from 'hds-react';
```

Comment:
The selected topics aren't added to the selections list. The location filter implementation in `containers/SelectionsContainer.tsx` should match this pretty closely.

Reply:
Good catch, thanks! I was not familiar with this react code before so I missed `SelectionsContainer`.